packages/scripts/README.md

```diff
@@ -30,6 +30,10 @@ This is how you execute those scripts using the presented setup:
 
 ## Available Scripts
 
+### `wp-scripts lint-pkg-json`
+
+Helps enforce standards for your package.json file. It uses [npm-package-json-lint](https://www.npmjs.com/package/npm-package-json-lint) with the set of default rules provided. You can override them with your own rules as described in [npm-package-json-lint wiki](https://github.com/tclindner/npm-package-json-lint/wiki).
+
```

Comment:
Let's include an example here:

```json
  "scripts": {
    "lint:pkg-json": "wp-scripts lint-pkg-json ."
  },
```

Without the `.` at the end of the command this error is displayed: `No lint targets provided`

Reply:
Yes, good call. We can make sure it is provided as default instead. What do you think?

Reply:
By default is a good idea if it can be overridden using `packages/` for example with Lerna repos.

Regardless of that though we should include some basic docs here in the readme

Reply:
Yes, let's start with your example and iterate :)